plugins/woocommerce-admin/client/customize-store/style.scss

```diff
@@ -77,7 +82,7 @@ body.woocommerce-customize-store.js.is-fullscreen-mode {
 		padding: 12px;
 		width: 404px;
 		border-radius: 2px;
-		background: var(--gutenberg-transparent-blueberry, rgba(56, 88, 233, 0.04));
+		background: var(--wp-admin-theme-color-background-04, rgba(168, 168, 170, 0.301));
```

Comment:
Should we use `rgba(56, 88, 233, 0.04)` for fallback? Or can you point out where `rgba(168, 168, 170, 0.301)` comes from? 😅

Reply:
I just picked a grey that would work with any theme colour 🫣

Reply:
Ok, make sense 😆